nbclient/client.py

```diff
@@ -51,7 +55,7 @@ class NotebookClient(LoggingConfigurable):
         ),
     ).tag(config=True)
 
-    timeout_func = Any(
+    timeout_func: t.Any = Any(
```

Comment:
Usually you do from `typing import Any`, but this is fine

Reply:
Yes but some names in `typing` collide with `traitlets` (`List`, `Any`, `Dict`), it's annoying.

Reply:
Oh... yeah good point